### DISCUSSION TITLE
how easily can the wikiaccess on github be ?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#55176] [New to GitHub]
Hello and good day, dear community

Wiki page on Github - how easily can I set up editing here...!?

Is it possible to edit here without an account?

I have a project – in which I want to get people involved – and I need very low-threshold access.

Confluence is too expensive for me

Miro as well - and the projects since on Github - they already exist anyway.

AFFiNE is great in my opinion - but not quite that far developed at the moment.

Also - if necessary, it would help me if you can easily edit and write on the Github wiki - very low-threshold. What do you mean!?

**Comment by airtower-luna** [comment]
Editing will require an account. You can choose whether to allow any user or only collaborators in the repository settings, right below where you choose if you want to have a wiki at all.

![grafik](https://github.com/community/community/assets/5814919/e84a795e-d328-4d7f-98f1-31edaf7a9f43)
